airbyte-integrations/connector-templates/destination-python/metadata.yaml.hbs

```diff
@@ -7,6 +7,10 @@ data:
       enabled: false
     cloud:
       enabled: false
+  connectorBuildOptions:
+    # Please update to the latest version of the connector base image.
+    # https://hub.docker.com/r/airbyte/python-connector-base
+    baseImage: docker.io/airbyte/python-connector-base:1.0.0@sha256:dd17e347fbda94f7c3abff539be298a65af2d7fc27a307d89297df1081a45c27
```

Comment:
Should we use the tags like this long? should the `python-connector-base:<version>` be enough to proceed with the base image?

Reply:
We want to encourage the use of "long" tags with sha for build reproducibility, it's the only way to uniquely identify an image in dockerhub

Reply:
Then the corresponding instructions should be given, as well as pattern matching to the `metadata.yaml` parser so we could catch the incorrect input here.  WDYT?

Reply:
Will do, metadata validation will come next. https://github.com/airbytehq/airbyte/issues/30234